src/Admin.js

```diff
@@ -151,4 +153,9 @@ Admin.propTypes = {
     initialState: PropTypes.object,
 };
 
-export default Admin;
+export default withContext(
+    {
+        authClient: PropTypes.func,
```

Comment:
Done

![screen](http://i.imgur.com/G112Ts9.png)

It works for the other components as well